test/unit/menu.test.js

```diff
@@ -18,4 +19,44 @@ test('should place title list item into ul', function() {
   ok(titleElement.innerHTML === 'TestTitle', 'title element placed in ul');
 
   player.dispose();
-});
\ No newline at end of file
+});
+
+q.test('clicking should display the menu', function() {
+  expect(6);
+
+  var player, menuButton, el;
```

Comment:
should be 3 `let`s.
We really need to update videojs to use `vjsstandard` already so I won't have to keep pointing these out myself :)